### ISSUE TITLE
Critical Dependency Vulnerability: mocha > growl

### ISSUE THREAD

**Issue by emmercm** [ubenzer/metalsmith-eslint#43]
From `npm audit`:

```
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Critical      │ Command Injection                                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ growl                                                        │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=1.10.2                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ metalsmith-eslint                                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ metalsmith-eslint > mocha > growl                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/146                             │
└───────────────┴──────────────────────────────────────────────────────────────┘
```

**Comment by ubenzer** [comment]
Released v4.0.0 with updated dependencies. Thanks for letting me know!

**Comment by emmercm** [comment]
Thanks so much @ubenzer!
